src/popup/components/InputField.vue

```diff
@@ -128,59 +148,62 @@ export default {
       default: null,
     },
   },
-  data: () => ({
-    focused: false,
-  }),
-  computed: {
-    inputId() {
-      return `input-${this._uid}`;
-    },
-    inputMode() {
-      return this.type === 'number' ? 'decimal' : 'text';
-    },
-    hasError() {
-      if (typeof this.message === 'object') {
-        return this.message?.status === 'error';
-      }
-      return !!this.message;
-    },
-    hasWarning() {
-      if (typeof this.message === 'object') {
-        return this.message?.status === 'warning';
-      }
-      return false;
-    },
-    showMessage() {
-      if (typeof this.message === 'object') {
-        return !this.message?.hideMessage;
-      }
-      return !!this.message;
-    },
-    availableTextLimit() {
-      return (this.textLimit && this.value?.length)
-        ? this.textLimit - this.value.length
-        : this.textLimit;
-    },
-  },
-  methods: {
-    checkIfNumber(event) {
+  setup(props, { emit }) {
+    const focused = ref(false);
+
+    const inputId = `input-${getCurrentInstance()?.uid}`;
+    const inputMode = computed(() => props.type === 'number' ? 'decimal' : 'text');
+    const hasError = computed(
+      () => (typeof props.message === 'object')
+        ? props.message?.status === 'error'
+        : !!props.message,
+    );
+    const hasWarning = computed(
+      () => (typeof props.message === 'object')
+        ? props.message?.status === 'warning'
```

Comment:
You have declared a variable for statuses here
https://github.com/aeternity/superhero-wallet/pull/1772/files#diff-d176ac880b039692a793823c27562dd25951237c941c78490de5fc64e8eb6f82R114

Reply:
True, changed.